documentation/CAMARA-Security-Interoperability.md

```diff
@@ -279,6 +290,175 @@ Camara recommends that implementations run the OIDF interoperability suite and a
 * [RFC 7519 - JSON Web Token (JWT)](https://www.rfc-editor.org/info/rfc7519)
 * [RFC 7521 - Assertion Framework for OAuth 2.0 Client Authentication and Authorization Grants](https://www.rfc-editor.org/info/rfc7521)
 * [RFC 7523 - JSON Web Token (JWT) Profile for OAuth 2.0 Client Authentication and Authorization Grants](https://www.rfc-editor.org/info/rfc7523)
+* [RFC 7636 - Proof Key for Code Exchange by OAuth Public Clients](https://www.rfc-editor.org/info/rfc7636)
 * [RFC 8259 - The JavaScript Object Notation (JSON) Data Interchange Format](https://www.rfc-editor.org/info/rfc8259)
 * [RFC 8414 - OAuth 2.0 Authorization Server Metadata](https://www.rfc-editor.org/info/rfc8414)
 
+## Appendix A: Error Responses
+
+This section describes the error responses that the Authorization Server MUST return for all flows defined in this profile. Based on the supported functionality, the set of errors and the scenarios that can generate them are defined from the specifications already referenced in this document.
+
+### Authentication Error Response
+
+#### Authorization Code Flow
+
+If the request fails due to a missing, invalid, or mismatching redirection URI, or if the client identifier is missing or invalid,
+the authorization server MUST NOT automatically redirect the user-agent and SHOULD inform the resource owner of the
```

Comment:
As far as I know, the only suggestion ever made was to send the request object by value, not by reference. So this error would not occur as it would return `request_uri_not_supported`.

Reply:
@garciasolero 
I don't understand what an invalid request_uri has to do with whether a request object is send by value or reference.

Common invalid redirect_uri cases are:
- value is not matching an registered values for the client_id
  e.g. redirect_uri=https://attacker.example.com/ with a registered value being https://client.example.com/
  e.g. redirect_uri=ts43client: with a registered value being https://ts43client.example.com/
In these cases the OP MUST not use https://attacker.example.com/ or ts43client in any way but must respond with an HTTP 400 Bad Request

Reply:
Sorry, Axel, but I don't understand what you mean. Initially, you talked about the `invalid_request_uri` error related to passing the request object by reference (`request_uri` parameter), and now you're talking about the `redirect_uri` parameter, which is unrelated.